### PR TITLE
Feature/intake form dark mode

### PR DESCRIPTION
**Pull request by Valeriusdev**
05/09/22

What work was done?
Converted the Intake Form to Dark Mode.

Loom video: https://www.loom.com/share/3dba5ba9728b4ce5b56c2b22564dbd7d
Trello card: https://trello.com/c/aiqPKzJ8/67-convert-intake-form-to-dark-mode

### CODE REVIEW

**Review by MinaMonsi — COMMENTED** [review]
Great work! I have questions about lines 19 through 60 of uncommented code.  Shouldn't that be deleted if not in use? Or have you been instructed to leave it there?

**Comment by Valeriusdev** [comment]
Thank you for your feedback @MinaMonsi. I will ask our Release Manager if we will need the uncommented code in the future.

**Comment by Valeriusdev** [comment]
@MinaMonsi Our Release Manager told me that it's ok to leave the commented out code there for now.

**Comment by aurelions** [comment]
Looks good to me, the only thing I saw was actually already answered in the exchange between Marina and yourself!

**Review by aurelions — APPROVED** [review]
After watching the loom video I can see everything was carefully thought out and done very well!

**Review by MinaMonsi — APPROVED** [review]
Thank you for clarifying.  Great work on the intake form.

**Review by ashtilawat23 — APPROVED** [review]
Great atomic PR! Only 2 files with >200 lines of code. 
Only suggestion: 
- The title of the PR should not be the same as your branch name. 
- Take some time to change the title to something more specific about the code you are actually writing.

**Comment by SJMucho** [comment]
Looks great! Nice job!

**Review by Mad-Mal — COMMENTED** [review]
Great job Val! This is very clean, easy to read and understand. I had the same question as Marina but it looks like its already been answered!

File: src/components/common/ClientIntakeForm/IntakeForms/ClientFamilyInfo.js [hunk 1]
```diff
@@ -16,48 +16,48 @@ import ChildFamilyMember from './FamilyMembers/ChildFamilyMember';
...
-    justifyContent: 'space-between',
-    margin: '0 10px',
-  };
-
-  const checkboxStyles = {
-    display: 'flex',
-    flexDirection: 'column',
-    flexWrap: 'wrap',
-  };
+  // const subsectionContainer = {
```
Comment (tinomen) [inline src/components/common/ClientIntakeForm/IntakeForms/ClientFamilyInfo.js:19]: Whoa, this is a lot of commented out code! Do we need to leave this in.

**Review by tinomen — COMMENTED** [review]
just the 1 thing, great otherwise

**Pull request merged**